src/queue.c

```diff
@@ -6509,6 +6511,8 @@ _dispatch_runloop_queue_set_handle(dispatch_lane_t dq,
 #elif defined(__linux__)
 	// encode: 0 is a valid fd, so offset by 1 to distinguish from NULL
 	dq->do_ctxt = (void *)(uintptr_t)(handle + 1);
+#elif defined(__FreeBSD__)
```

Comment:
Do we need this? It looks like it's the same as the `defined(__unix__)` case below, which should capture on FreeBSD.